### ISSUE TITLE
Adding new Vimeo video error

### ISSUE THREAD

**Issue by Michael-073** [magento/magento2#32897]
I noticed that there are some videos from Vimeo that will return different data. Depending on what data is returned, the script that processes the video can crash.

### Preconditions (*)
1. Magento version 2.4.0
1. Patch MDVA-35092 applied

### Steps to reproduce (*)
1. Go to Admin -> Catalog ->Products
1. Select a product to edit
1. Go to Images and Videos section. Click Add Video
1. In the Url field, use a video url from Vimeo that is missing required information  ( example: https://vimeo.com/313826626 )

### Expected result (*)
1. The video information to be filled in and the save button to be enabled.

### Actual result (*)
1. No other fields are filled in and the save button remains disabled
1. The following appears in the web browser console
 ``` 
Uncaught TypeError: Cannot read property '0' of null
    at $.<computed].[computed]._formatVimeoDuration (get-video-information.js:592)
    at $.[computed].[computed]._formatVimeoDuration (jquery-ui-1.9.2.js:432)
    at $.[computed].[computed]._onVimeoLoaded (get-video-information.js:502)
    at Object.proxy [as success] (jquery.js:535)
    at fire (jquery.js:3238)
    at Object.fireWith [as resolveWith> (jquery.js:3368)
    at done (jquery.js:9846)
    at HTMLScriptElement.script.onload.script.onreadystatechange (jquery.js:10436)
```

The `_formatVimeoDuration` function in the `get-video-information.js` file has a parameter called `seconds`. The problem is that seconds is undefined. This is a result of the data returned from Vimeo missing the duration property. 

Another error can occur when the description for the video is missing as well

## 
Please provide [Severity](https://devdocs.magento.com/guides/v2.4/contributor-guide/contributing.html#backlog) assessment for the Issue as Reporter. This information will help during Confirmation and Issue triage processes.

- < ] Severity: **S0** *- Affects critical data or functionality and leaves users without workaround.*
- [ ] Severity: **S1** *- Affects critical data or functionality and forces users to employ a workaround.*
- [ ] Severity: **S2** *- Affects non-critical data or functionality and forces users to employ a workaround.*
- [x] Severity: **S3** *- Affects non-critical data or functionality and does not force users to employ a workaround.*
- [ > Severity: **S4** *- Affects aesthetics, professional look and feel, “quality” or “usability”.*

**Comment by mrtuvn** [comment]
Can you reproduce in latest code base 2.4-develop ? Since i have checked some change in api endpoint vimeo and also some modify in codebase too

**Comment by magento-engcom-team** [comment]
@engcom-Oscar Thank you for verifying the issue.

Unfortunately, not enough information was provided to acknowledge ticket. Please consider adding the following:
- [ ] Add `"Reproduced on "` label(s) to this ticket based on verification result

Once all required information is added, please add label `"Issue: Confirmed"` again.
Thanks!

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Oscar 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-42105` were created

**Issue Available**: @engcom-Oscar, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by magento-engcom-team** [comment]
Hi @Michael-073. 

Thank you for your report and collaboration!

The related internal Jira ticket `MC-42105` was closed as `Fixed`. 

The fix will be available with the upcoming `2.4.3` release.
